joomla-projects/custom-fields
# Add new core rule core.edit.value

Pull Request for Issue #46.
#### Summary of Changes

Adding a new core rule core.edit.value. As the core had no rule to define on a per field basis who can edit the value, DPFields had to do a workaround for it's field editing rule. This PR removes the workaround and introduces a new core rule core.edit.value. This rule can be used then in the future on other components when they want to have a per field permission and not only per item.
#### Testing Instructions
- Create a field and set the edit value permission for Registered to _None_.
- Create an article and set the edit permission for Registered to _allowed_.
- Edit the article on the front.

The expected result is that the field should be disabled.
